option.go

```diff
@@ -282,6 +302,61 @@ func (option *Option) Set(value *string) error {
 	return convert("", option.value, option.tag)
 }
 
+// Set the value of a "terminated" option to the specified value.
+// An error will be returned if the specified value could not be
+// converted to the corresponding option value type.
+// If option value is slice of slice, the parameter is converted
+// as a slice and appended to the option value. Otherwise, if
+// option value is slice, option value is replaced by parameter.
```

Comment:
```suggestion
// If the option value is a slice of slices, each element in the parameter list
// (excluding the terminator) is converted and the resulting list is appended to the option value.
// Otherwise, each element is converted and the option value is set to
// the resulting list.
```